src/status_im/ui/screens/wallet/account/views.cljs

```diff
@@ -119,13 +125,64 @@
          (= tab :history)
          [transactions address])])))
 
+(views/defview bottom-send-recv-buttons [{:keys [address type] :as account} anim-y]
+  [react/animated-view {:style {:background-color colors/white
+                                :bottom           0
+                                :flex-direction   :row
+                                :height           button-group-height
+                                :position         :absolute
+                                :shadow-offset    {:width 0 :height 1}
+                                :shadow-opacity   0.75
+                                :shadow-radius    1
+                                :transform        [{:translateY anim-y}]
+                                :width            "100%"}}
+   (if (= type :watch)
+     [react/view {:flex 1 :align-items :center :justify-content :center}
+      [react/text {:style {:margin-left 8 :color colors/blue-persist}}
+       (i18n/label :t/watch-only)]]
+     [button
+      (i18n/label :t/wallet-send)
+      :main-icons/send
+      colors/blue-persist
+      #(re-frame/dispatch [:wallet/prepare-transaction-from-wallet account])])
+   [button
+    (i18n/label :t/receive)
+    :main-icons/receive
+    colors/blue-persist
+    #(re-frame/dispatch [:show-popover {:view :share-account :address address}])]])
+
+(defn anim-listener [anim-y scroll-y]
+  (let [to-show (atom false)]
+    (animation/add-listener
+     scroll-y
+     (fn [anim]
+       (let [y-trigger 149]
+         (cond
+           (and (>= (.-value anim) y-trigger) (not @to-show))
+           (animation/start
+            (styles/bottom-send-recv-buttons-raise anim-y)
+            #(reset! to-show true))
+
+           (and (< (.-value anim) y-trigger) @to-show)
+           (animation/start
+            (styles/bottom-send-recv-buttons-lower anim-y button-group-height)
+            #(reset! to-show false))))))))
+
 (views/defview account []
   (views/letsubs [{:keys [name address] :as account} [:multiaccount/current-account]]
-    [react/view {:flex 1 :background-color colors/white}
-     [toolbar-view name]
-     [react/scroll-view
-      [react/view {:padding-left 16}
-       [react/scroll-view {:horizontal true}
-        [react/view {:flex-direction :row :padding-top 8 :padding-bottom 12}
-         [account-card account]]]]
-      [assets-and-collections address]]]))
+    (let [anim-y (animation/create-value button-group-height)
+          scroll-y (animation/create-value 0)]
+      (anim-listener anim-y scroll-y)
+      [react/view {:flex 1 :background-color colors/white}
+       [toolbar-view name]
+       [react/animated-scroll-view
+        {:on-scroll (animation/event
+                     [{:nativeEvent {:contentOffset {:y scroll-y}}}]
+                     {:useNativeDriver true})
+         :scrollEventThrottle 1}
```

Comment:
```suggestion
		 :contentContainerStyle {:padding-bottom button-group-height}
         :scrollEventThrottle 1}
```

This change can be added here to solve the problem of covering. Better to use here, because it is directly connected with this scroll view which adds the bottom bars. Also, it will add the correct spacing to any new element that can be to this scroll-view, and will be controlled by the same variable as height of the bar.

Reply:
Thanks @Ferossgp!  I dropped my WIP commit and applied the change you suggested, it works great.